static/js/ReaderApp.jsx

```diff
@@ -1434,7 +1434,8 @@ toggleSignUpModal(modalContentKind = SignUpModalKind.Default) {
     this.state.panels = []; // temporarily clear panels directly in state, set properly with setState in openPanelAt
     this.openPanelAt(0, ref, currVersions, options);
   }
-  openPanelAt(n, ref, currVersions, options, replace, convertCommentaryRefToBaseRef=true, replaceHistory=false, saveLastPlace=true) {
+  openPanelAt(n, ref, currVersions, options, replace, convertCommentaryRefToBaseRef=true,
+              replaceHistory=false, saveLastPlace=true, openCommentarySidePanel=false) {
```

Comment:
I think the way this parameter is currently called is confusing and the docs for it are also confusing because it's stated in the negative and there are multiple if statements in the description of the parameter.

Maybe instead call this `forceOpenCommentaryPanel` and default is `false`. Docs can say something like `when true, this will open the commentary side panel with this ref in it regardless of the refs depth. When false, will only open side panel if ref is depth 3 or greater`.